src/components/daily/bolustooltip/BolusTooltip.js

```diff
@@ -172,10 +173,48 @@ class BolusTooltip extends React.Component {
     return extendedLine;
   }
 
+  getFormatedInputTime(bolus, timePrefs) {
+    const time = formatLocalizedFromUTC(bolus.inputTime, timePrefs, getHourMinuteFormat());
+    return time;
+  }
+
+  getBolusTypeLine(bolusType) {
+    return bolusType && (
+      <div className={styles.bolus}>
+        <div className={styles.label}>{t('Bolus')}</div>
+        <div className={styles.value}>{t(bolusType)}</div>
+      </div>
+      );
```

Comment:
```suggestion
    );
```

or if using the previous suggestion:
```suggestion
    ) : null;
```